bundles/CoreBundle/Resources/config/services.yaml

```diff
@@ -215,9 +215,9 @@ services:
         public: true
         class: Pimcore\Model\DataObject\QuantityValue\DefaultConverter
 
-    Pimcore\Bundle\CoreBundle\Request\ParamConverter\DataObjectParamConverter:
+    Pimcore\Bundle\CoreBundle\Request\ParamConverter\DataObjectParamResolver:
```

Comment:
Mhh, shouldn't we keep them both in 10.6? 
I see that DataObjectParamConverter is marked as internal but in the custom routes docs it is also telling that it could be used.

Reply:
`@ParamConverter` comes SensioBundleExtraBundle so it still works and the resolving is done internally so it shouldn't be a problem.